utils/project/path.go

```diff
@@ -66,11 +66,19 @@ func GoPath(directory string) (goPath string, absPath string, err error) {
 
 // PackageForPath gets package path for a path.
 func PackageForPath(directory string) (string, error) {
-	goPath, absPath, err := GoPath(directory)
+	absPath, err := filepath.Abs(directory)
```

Comment:
Should we update the test `TestPackageForPath`
https://github.com/caicloud/nirvana/blob/f63d17bd5cceeb954650a2a98b38da1ead8b94f8/utils/project/path_test.go#L23
to include the case that `directory` is outside GOPATH?

Reply:
Updated, PTAL